app/models/subscription_target.rb

```diff
@@ -0,0 +1,24 @@
+class SubscriptionTarget
+  include ApiEntity
```

Comment:
This gives you the header method:
```suggestion 
  include AuthenticatableApiEntity
```